src/PageController.php

```diff
@@ -461,13 +466,15 @@ public function register_page( $options ) {
 			);
 
 			if ( method_exists( '\Automattic\WooCommerce\Navigation\Menu', 'add_item' ) ) {
+				$top_level_ids = array( 'woocommerce-home', 'woocommerce-analytics-customers' );
 				\Automattic\WooCommerce\Navigation\Menu::add_item(
 					array(
-						'id'         => $options['id'],
-						'parent'     => $options['parent'],
-						'title'      => $options['title'],
-						'capability' => $options['capability'],
-						'url'        => $options['path'],
+						'id'                => $options['id'],
+						'parent'            => $options['parent'],
+						'title'             => $options['title'],
+						'capability'        => $options['capability'],
+						'url'               => $options['path'],
+						'is_top_level_item' => in_array( $options['id'], $top_level_ids, true ),
```

Comment:
Quick note that this will need to be updated if the change is made in https://github.com/woocommerce/navigation/pull/117#discussion_r500223867